.vscode/settings.json

```diff
@@ -8,4 +8,4 @@
     },
     // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
     "typescript.tsc.autoDetect": "off"
-}
+}
```

Comment:
Can we not include this because it has no changes ;~; ?

Reply:
I'm not sure why there's a difference there.. do you know what it is?

Reply:
oops, it's a newline... I'll look into it!